README.md

```diff
@@ -258,11 +258,15 @@ function handleEditChange(updatedData) {
 onEditChange(initialData, handleEditChange);
 ```
 
-### Level 4: Enable Managing Blocks directly on your frontend ([TODO](https://github.com/collective/volto-hydra/issues/4))
+### Level 4: Enable Managing Blocks directly on your frontend
 
-If you completed level 2 & 3 (made blocks clickable and enabled live updates) then the editor will automatically gain the management of blocks on the frontend using the quanta toolbar
-- Add blocks ([TODO](https://github.com/collective/volto-hydra/issues/27))
-- remove blocks ([TODO](https://github.com/collective/volto-hydra/issues/26))
+If you completed level 2 & 3 (made blocks clickable and enabled live updates) then the editor will automatically gain the management of blocks on the frontend using the quanta toolbar.
+
+With Quanta toobar, you can use following features:
+
+- You can click on '+' Icon (appears at the bottom-right of the container in which you added `data-bloc-uid="<<BLOCK_UID>>>"` attribute) to add a block below the current block by choosing a type from BlockChooser popup.
```

Comment:
if you look at the screen shot I think he also puts a shadow on the add menu and moves it out from the right and bottom edges somewhat to make it stand out a bit more

Reply:
I'll split out the behavour for mobile into other cards

Reply:
> if you look at the screen shot I think he also puts a shadow on the add menu and moves it out from the right and bottom edges somewhat to make it stand out a bit more

You mean BlockChooser popup? if yes, then it already got a decent box-shadow tho.. we can try out more if you want to check it out,
and if you mean the 'plus' icon, then it already a nice blue border keeping it distinctive.

Reply:
yes blockchooser. if its got a shadow then maybe it should needs to be moved out from the edges of the iframe a bit more.

Reply:
Will this much works?

![Screenshot from 2024-06-29 08-09-15](https://github.com/collective/volto-hydra/assets/99530996/d017d5d4-0f99-4f47-b43a-64ea21dcb9f6)